documentation/source/rescap.rst

```diff
@@ -0,0 +1,120 @@
+###############################
+The cocotb ``rescap`` Testbench
+###############################
+
+.. versionadded:: 1.5
+
+This is the testbench :mod:`test_rescap` for the design ``rescap`` showing
+how cocotb can be used in an analog-mixed signal (AMS) simulation.
+
+******************
+Overview Schematic
+******************
+
+.. image:: rescap_schematic.png
+
+
+**********
+The Design
+**********
+
+The design consists of a resistor and capacitor model (both written in Verilog-AMS) connected
+in series in a SystemVerilog module, as shown here:
+
+.. code-block:: none
+
+          i_resistor             i_capacitor
+  vdd >---o--v^v^v--o------------o---||---o---< vss
+          p         n    vout    p        n
+
+
+The code of the design is:
+
+.. literalinclude:: ../../examples/mixed_signal/hdl/rescap.sv
+   :caption: rescap.sv
+   :language: systemverilog
+   :start-at: // the design-under-test
+
+
+*************
+The Testbench
+*************
+
+The testbench consists of both an :term:`HDL` part and a Python/cocotb part.
+While having :term:`HDL` as part of the testbench is not common with cocotb, it is perfectly possible.
```

Comment:
I'm wondering if we need to link HDL every time we use the term, or maybe just once per section, it's somewhat distracting. Very small nit, you can ignore.